src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -162,7 +162,7 @@ private void WriteOutputInBackground(bool isError, string firstLine, string? sec
             // without deadlocks with other operations.
             // This could happen, for example, for writers provided by the Interactive Window,
             // and in the case where the window is being disposed.
-            Task.Run(() =>
+            await Task.Run(() =>
```

Comment:
>await [](start = 12, length = 5)

Could this cause a problem that's described in the comment above?

Reply:
Only if the `Process.Exited` event is fired on the main thread.

Reply:
Isn't that possible?

Reply:
Not sure currently. I'll think about alternate ways to handle this.